src/relay/backend/build_module.cc

```diff
@@ -460,12 +460,11 @@ class RelayBuildModule : public runtime::ModuleNode {
     ret_.params = graph_codegen_->GetParams();
 
     auto lowered_funcs = graph_codegen_->GetLoweredFunc();
-    if (lowered_funcs.size() != 0) {
-      ret_.mod = tvm::build(
-        lowered_funcs,
-        target_host_,
-        BuildConfig::Current());
-    }
+    CHECK(lowered_funcs.size() != 0) << "no lowered funcs exist in the compiled module";
```

Comment:
I see.  I wasn't sure if there were cases where one would want to use a build result without a defined module.  Maybe the right change would be to modify the [ModuleNode* operator->](https://github.com/dmlc/tvm/blob/master/include/tvm/runtime/module.h#L183) methods to check that the shared ptr isn't null?

Reply:
I personally think returning a nullptr is okay. Dereferencing a nullptr should crash. I would recommend logging a warning here so that users would know the module is empty.

Reply:
Yeah.  A warning seems like a good compromise.